docs/source/index.rst

```diff
@@ -41,34 +55,34 @@ The following commands will download and build UCX v1.6 :doc:`release <download>
     $ ./contrib/configure-release --prefix=$PWD/install
     $ make -j8 install
 
+    $ gcc test/examples/ucp_client_server.c -lucp -lucs -o ucp_client_server \
+          -Iinstall/include -Linstall/lib
+    $ export LD_LIBRARY_PATH=$PWD/instal/lib
+    $ ./ucp_client_server &
+    $ ./ucp_client_server -a <ip-address>
```

Comment:
maybe change ip-address to $(hostname -i) so it will work from copy&paste?

Reply:
then it will use TCP, and we want to demo RDMA..

Reply:
what will make use rdma? IPoIB i/f IP?

Reply:
yes, passing IPoIB or RoCE network interface address